docs/preact-testing-library/intro.md

```diff
@@ -0,0 +1,65 @@
+---
+id: intro
+title: Preact Testing Library
+sidebar_label: Introduction
+---
+
+[`react-testing-library`][gh] builds on top of `dom-testing-library` by adding
+APIs for working with Preact components.
+
+```
+npm install --save-dev preact-testing-library
+```
+
+- [preact-testing-library on GitHub][gh]
+
+[gh]: https://github.com/antsmartian/preact-testing-library
+
+## The problem
+
+You want to write maintainable tests for your Preact components. As a part of
```

Comment:
This page could probably be a little more lightweight by referring to the home page and the React pages - this one is a little less likely to be updated promptly if it duplicates a lot of the content from another page